docs/home/HOWTO.md

```diff
@@ -147,6 +149,25 @@ sudo -u postgres -i
 And then configure database server using psql according to your needs and
 PostgreSQL documentation, to which link you can find at <https://www.postgresql.org/docs/>
 
+### How to configure PostgreSQL replication
+
+In order to configure PostgreSQL replication to your data.yaml add block similar to the one below to core section:
```

Comment:
I think this would sound better: In order to configure PostgreSQL replication add to your data.yaml a block similar...